src/navigator/NavigatorWrapper.tsx

```diff
@@ -81,6 +83,12 @@ export const NavigatorWrapper = () => {
   const shouldForceBackup = useSelector(shouldForceBackupSelector)
   const doingBackupFlow = useSelector(doingBackupFlowSelector)
 
+  React.useEffect(() => {
+    if (isUserInSanctionedCountry) {
+      navigate(Screens.SanctionedCountryErrorScreen)
+    }
+  }, [])
```

Comment:
nit: in case this information is not available / incorrect at the time of this component mounting, we should add the `isUserInSanctionedCountry` in the dependency array here :)

Reply:
I just removed `useEffect` implementation because users can bypass the error screen by sliding back...